components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -1737,6 +1737,131 @@ public static NativeObject jsFunction_getAllPaginatedPublishedAPIs(Context cx, S
 
     }
 
+    public static NativeObject jsFunction_getAllPaginatedPublishedLightWeightAPIs(Context cx, Scriptable thisObj,
```

Comment:
Shall we add java docs to this method?

Reply:
Done in ccca5d254427d1d6d57b5c373cdf746ec51dc6e4